tests/5.0/target/test_target_defaultmap_to_from_tofrom.c

```diff
@@ -0,0 +1,144 @@
+//===--- test_target_defaultmap_to_from_tofrom.c -----------------------------------------------------------===//
+//
+//  OpenMP API Version 5.0 Nov 2018
+//
+//  This test checks behavior of the defaultmap clause when the specified implicit-behavior  
+//  is to, from, and tofrom. The variable-categories avaiable for defaultmap are scalar, aggregate, 
+//  and pointer.
+//
+//
+////===-------------------------------------------------------------------------------------------===//
```

Comment:
Might want to shorten the line lengths in this header comment. (82 characters is our usual)